src/app/business/block/hosts.component.ts

```diff
@@ -0,0 +1,306 @@
+              "tenantId": "78fe0bb5-06ca-4e77-8465-ad363ffd8ea3",
+              "hostName": "Hardy",
+              "osType": "Linux",
+              "accessMode": "Agentless",
+              "ip": "19.9.38.184",
+              "port": 8775,
+              "availabilityZones": [
+                "az3",
+                "az3",
+                "az2"
+              ],
+              "initiators": [
+                {
+                  "portName": "qui1",
+                  "protocol": "SCSI"
+                },
+                {
+                  "portName": "labore2",
+                  "protocol": "FC"
+                }
+              ],
+              "isActive": true
+            },
+            {
+              "id": "5dc533ad2f20343932e4aa7d",
+              "createdAt": "2019-07-0802:07:SS.-06:-30-06:-30",
+              "updatedAt": "2019-08-2615:08:SS.-06:-30-06:-30",
+              "tenantId": "d0c9f534-a9fe-4078-a328-3e3b87bf6010",
+              "hostName": "Aisha",
+              "osType": "Windows",
+              "accessMode": "Agentless",
+              "ip": "6.134.249.109",
+              "port": 22504,
+              "availabilityZones": [
+                "az1",
+                "az1"
+              ],
+              "initiators": [
+                {
+                  "portName": "magna1",
+                  "protocol": "SCSI"
+                },
+                {
+                  "portName": "amet2",
+                  "protocol": "NVMe"
+                }
+              ],
+              "isActive": true
+            }
+          ];
+    }
+
+   deleteHost(id){
+       console.log("Delete host", id);
+       this.HostsService.deleteHost(id).subscribe((res)=>{
+        this.msgs.push({severity: 'success', summary: 'Success', detail: 'Host Deleted Successfully.'});
+        this.router.navigate(['/block',"fromHosts"]);
+    }, (error) =>{
+        this.msgs = [];
+        this.msgs.push({severity: 'error', summary: 'Error', detail: error.message});
+    });
+   }
+
+    batchDeleteHosts(hosts){
+        let arr=[], msg;
+        if(_.isArray(hosts)){
+            hosts.forEach((item,index)=> {
+                arr.push(item.id);
+            })
+            msg = "<div>Are you sure you want to delete the selected hosts?</div><h3>[ "+ hosts.length +" Hosts ]</h3>";
+        }else{
+            arr.push(hosts.id);
+            msg = "<div>Are you sure you want to delete the host?</div><h3>[ "+ hosts.hostName +" ]</h3>";
+        }
+
+        this.confirmationService.confirm({
+            message: msg,
+            header: this.I18N.keyID['sds_block_host_delete'],
+            acceptLabel: this.I18N.keyID['sds_block_volume_delete'],
+            isWarning: true,
+            accept: ()=>{
+                arr.forEach((item,index)=> {
+                    this.deleteHost(item)
+                })
+
+            },
+            reject:()=>{}
+        })
+
+    }
+
+
+  /*   modifyVolume() {
```

Comment:
This is not relevant, right? If yes, please remove this

Reply:
ok.